aurpublish.in

```diff
@@ -137,6 +144,11 @@ if (( LOG )); then
     exit 0
 fi
 
+if [[ "$url_set" != "true" ]]; then
```

Comment:
For flag variables, the aurpublish style is to use `URL_SET=1` and check with `(( URL_SET ))`, see how log, speedup, pull, setup work.

Reply:
Fixed!

Note that is is a negated test, so the final version is not quite as pretty as the others.